generator/src/test/java/com/scottlogic/deg/generator/generation/combinationstrategies/ExhaustiveCombinationStrategyTests.java

```diff
@@ -34,8 +34,8 @@ void beforeEach(){
     @Test
```

Comment:
If possible, it would be good to add a regression test for this bug here.

Reply:
good point, i'll do that